ReactAndroid/src/main/java/com/facebook/react/animated/FrameBasedAnimationDriver.java

```diff
@@ -60,7 +68,7 @@ public void runAnimationStep(long frameTimeNanos) {
     if (frameIndex >= mFrames.length - 1) {
       nextValue = mToValue;
       if (mIterations == -1 || mCurrentLoop < mIterations) { // looping animation, return to start
-        mStartFrameTimeNanos = frameTimeNanos;
+        mStartFrameTimeNanos = frameTimeNanos + ((long) FRAME_TIME_MILLIS) * 1000000L;
```

Comment:
@axemclion @kmagiera I think this line could be the culprit. I guess it could be possible that mStartFrameTimeNanos is bigger than the curent clock time.

Reply:
Maybe this would be better as `mStartFrameTimeNanos = -1;`.

Reply:
I remember there was some other problem when I was setting it to -1 but can't remember what exactly. Will try that and send a new PR

Reply:
Ah, ok I remember now, I didn't want `mFromValue` to be updated with the next loop and that happens when mStartFrameTimeNanos is -1. Will add another check for this case and send a PR

Reply:
Submitted it here: https://github.com/facebook/react-native/pull/18061